src/Offer/CommandHandlers/AddLabelHandler.php

```diff
@@ -40,11 +40,11 @@ public function handle($command): void
         }
 
         $this->labelService->createLabelAggregateIfNew(
-            new LegacyLabelName((string) $command->getLabel()),
+            new LegacyLabelName($command->getLabel()->getName()->toString()),
```

Comment:
Changing `createLabelAggregateIfNew` will make this `LegacyLabelName` obsolete

Reply:
I will fix this in a separate PR to remove the legacy Label Objects